### DISCUSSION TITLE
Elementals are missing description about double damage dealed to Elemental of opposite type

### DISCUSSION THREAD

**Discussion by Branikolog** [ihhub/fheroes2#10292] [General]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

Fire elemental hits 2x to Water Elemental. Air elemental hits 2x to Earth Elemental.
This info is not present in Elemental description window.
![image](https://user-images.githubusercontent.com/55348946/153755302-539d1888-d7a8-4e17-8375-a148aa0d6dd5.png)

### Describe the possible solution

The most perfect solution is to expand this window to be able to add more text within three text lines.
We can move creature sprite to be able to add one more text line. 
As temporary solution, information of increased damage can be shown in battle log.

**Comment by ihhub** [comment]
Moving it to the Discussion section as I don't think we would like to go with the "perfect" solution.
